common/casus_belli_types/00_civil_war.txt

```diff
@@ -1206,6 +1206,8 @@ independence_war = {
 							has_innovation = innovation_chronicle_writing
```

Comment:
There are 2 arts you don't use:
```
gfx/interface/illustrations/men_at_arms_small/gunmen.dds
gfx/interface/illustrations/men_at_arms_big/gunmen.dds
```

Reply:
This would be because they link to the riflemen unit that is in the #380 branch, if you want I can remove these images from this branch